### PR TITLE
Announcement of JabRef 3.3

### PR DESCRIPTION
**Pull request by mlep**
- Just a first draft for you to edit.
- The author of the post should be specified (and probably not me, since the announcement is made in the name of JabRef team)
- Color should be changed.
- Once posted on the blog, it should be sent to the mailing list. Who is doing it?

### CODE REVIEW

File: _posts/2016-04-19-JabRef3.3.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+author: "A generic name and address for JabRef?"
+---
+
+We are pleased to announce the release of JabRef version 3.3!
+
+Since yesterday, it can be downloaded from [FossHub](http://www.fosshub.com/JabRef.html), the official JabRef download site.
+
+This release contains new and improved features such as:
+- a file-based BibTeX and BibLaTeX mode 
+- an improved importation from ArXiv and Ovid
```
Comment (simonharrer) [inline _posts/2016-04-19-JabRef3.3.md]: import instead of importation

File: _posts/2016-04-19-JabRef3.3.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- a better handling of PDF files
+- a better integration of OpenOffice/LibreOffice style file
```
Comment (simonharrer) [inline _posts/2016-04-19-JabRef3.3.md]: files

File: _posts/2016-04-19-JabRef3.3.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+- a faster search
+- an easier installation under windows
+- a translation into Swedish
+- an online help
```
Comment (simonharrer) [inline _posts/2016-04-19-JabRef3.3.md]: - improved support on Mac OS X

File: _posts/2016-04-19-JabRef3.3.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+color: white
+author: "A generic name and address for JabRef?"
+---
+
+We are pleased to announce the release of JabRef version 3.3!
+
+Since yesterday, it can be downloaded from [FossHub](http://www.fosshub.com/JabRef.html), the official JabRef download site.
+
+This release contains new and improved features such as:
+- a file-based BibTeX and BibLaTeX mode
```
Comment (oscargus) [inline _posts/2016-04-19-JabRef3.3.md]: Maybe skip the "a", so:

file-based BibTeX and BibLaTeX mode(s?)

**Comment by mlep** [comment]
Suggested changes done (thanks!).

Remaining issues:
- The author of the post should be specified (and probably not me, since the announcement is made in the name of JabRef team). What should it be?
- Color should be changed. By what?
- Once posted on the blog, it should be sent to the mailing list. Who is doing it?

**Comment by koppor** [comment]
author: same as first post - jabref team

color: should be alternating, not a color ful thing as the faq. :)

I can do it after merging

Mailinglist: I can do :)

**Comment by mlep** [comment]
@koppor: Done. It up to you know!

File: _posts/2016-04-19-JabRef3.3.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+---
+title: Release of JabRef 3.3
+id: JabRef3.3
+author: "[JabRef Developers](https://github.com/orgs/JabRef/teams/developers)"
+---
+
+We are pleased to announce the release of JabRef version 3.3!
+
+Since yesterday, it can be downloaded from [FossHub](http://www.fosshub.com/JabRef.html), the official JabRef download site.
```
Comment (matthiasgeiger) [inline _posts/2016-04-19-JabRef3.3.md]: I'd propose "Since 16-04-17" instead of "yesterday".

File: _posts/2016-04-19-JabRef3.3.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+- file-based BibTeX and BibLaTeX modes 
+- improved import from ArXiv and Ovid
+- better handling of PDF files
+- better integration of OpenOffice/LibreOffice style files
+- faster search
+- easier installation under windows
+- translation into Swedish
+- online help
+- improved support on Mac OS X
+
```
Comment (matthiasgeiger) [inline _posts/2016-04-19-JabRef3.3.md]: One additional feature should be mentioned: "- save actions: automatically format your entries on save"

File: _posts/2016-04-19-JabRef3.3.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+It also includes about 40 bug fixes and the removal of a few features (mostly outdated).
+Details are listed on [gitHub](https://github.com/JabRef/jabref/blob/master/CHANGELOG.md).
```
Comment (matthiasgeiger) [inline _posts/2016-04-19-JabRef3.3.md]: I'd link to https://github.com/JabRef/jabref/blob/v3.3/CHANGELOG.md - which is the fixed changelog for this release.

**Comment by matthiasgeiger** [comment]
Thanks @mlep for your draft! 

I have just posted some additional comments... :wink:

**Comment by mlep** [comment]
@matthiasgeiger Thank you for your suggestions. They are now included.

**Comment by matthiasgeiger** [comment]
Published on jabref.org - @koppor You can send the mail now :wink:

**Pull request merged**